package.json

```diff
@@ -356,6 +359,36 @@
         "category": "ECL",
         "title": "%Copy WUID%"
       },
+      {
+        "command": "hpccPlatform.browseECLWatch",
+        "category": "ECL",
+        "title": "%Browse ECL Watch%"
+      },
+      {
+        "command": "hpccPlatform.openResults",
+        "category": "ECL",
+        "title": "%Open Results%"
```

Comment:
Shoud this be "Browse Results"?  IOW does it open in an external browser, or does it activate the results view in vscode?

Reply:
Can you open a new ticket to "Browse WU Details" for WUs

Reply:
Issue for vscode-ecl